examples/tutorials/run_v230_all_sets.py

```diff
@@ -0,0 +1,95 @@
+import os
+from acme_diags.run import runner
+from acme_diags.parameter.core_parameter import CoreParameter
+from acme_diags.parameter.area_mean_time_series_parameter import AreaMeanTimeSeriesParameter
+from acme_diags.parameter.enso_diags_parameter import EnsoDiagsParameter
+from acme_diags.parameter.qbo_parameter import QboParameter
+from acme_diags.parameter.diurnal_cycle_parameter import DiurnalCycleParameter
+from acme_diags.parameter.streamflow_parameter import StreamflowParameter
+
+
+#Define data paths for obs
+input_prefix ='/global/cfs/cdirs/e3sm/acme_diags'
+obs_climo = '/global/cfs/cdirs/e3sm/acme_diags/obs_for_e3sm_diags/climatology'
+obs_ts = '/global/cfs/cdirs/e3sm/acme_diags/obs_for_e3sm_diags/time-series'
```

Comment:
It would be better to only have to change the `input_prefix` in one place:
```
input_prefix = '/global/cfs/cdirs/e3sm/acme_diags/obs_for_e3sm_diags'
obs_climo = os.path.join(input_prefix, 'climatology')
obs_ts = os.path.join(input_prefix, 'time-series')
```